lib/saml_idp/request.rb

```diff
@@ -90,18 +87,6 @@ def requested_authn_contexts
       end
     end
 
-    def requested_ial_authn_context
-      requested_authn_contexts.select do |classref|
-        IAL_PREFIX.match?(classref) || LOA_PREFIX.match?(classref)
-      end.first
-    end
-
-    def requested_aal_authn_context
-      requested_authn_contexts.select do |classref|
-        AAL_PREFIX.match?(classref)
-      end.first
-    end
-
     def requested_vtr_authn_contexts
```

Comment:
question (): This can be removed too? It will require adjusting the vtr values handler in the idp code to not raise an error and simply remove any invalid/unknown values from the list of values.

Thoughts?

Reply:
I thought about it, but decided against it as I felt that it should be dealt with when we remove all VOT code.